custom-plugins/fictive_studios/admin/migrations/create-tables.php

```diff
@@ -7,11 +7,13 @@ public function createTables()
     {
         $this->loadMigrations();
         create_templates_tables();
+        create_printing_areas_tables();
     }
 
     public function loadMigrations()
     {
          require_once PLUGIN_ROOT. 'admin/migrations/templates.php';
+         require_once PLUGIN_ROOT. 'admin/migrations/printing-areas.php';
```

Comment:
Can we use this ? 
plugin_dir_url( dirname( __FILE__ ) )

if it work remove the PLUGIN_ROOT from root of the plugin, we dont't need it.

Reply:
I will use `plugin_dir_path(__FILE__)` here